cli/Valet/Configuration.php

```diff
@@ -41,6 +41,8 @@ function install()
      */
     function createConfigurationDirectory()
     {
+        $this->files->ensureDirExists(preg_replace('~/valet$~', '', VALET_HOME_PATH), user());
```

Comment:
it took me a minute to figure out what was going on here. Is there any simpler way to trim this off the end? If not, that's OK, but just for future folks who get lost like me lol

Reply:
I considered just str_replace, but the regex with `$` forcing the end of the string felt safer.

Granted, it's unlikely there'd be multiple `/valet` instances in it, so str_replace is probably fine.
Which direction would you like us to take?

Reply:
ughghghghgh i want the clearer option that's less precise but i feel like it will bite us later. Let's keep it. <3